polaris.shopify.com/src/components/PatternPage/PatternPage.module.scss

```diff
@@ -127,29 +127,25 @@
 }
 
 .HowItHelps {
-  .ImageWrapper {
+  & > img {
     display: block;
     border: 1px solid var(--p-border-subdued);
-    position: relative;
-    flex: auto;
-    overflow: hidden;
     aspect-ratio: 361.9/180.9;
-    width: 100%;
     border-radius: 0.25rem;
     background: var(--p-surface-depressed);
-    > img {
-      width: 100%;
-      object-fit: fill;
-    }
```

Comment:
No longer need these styles because we're not using `next/image` (which complicated things)